Task 5.2/Task5.2/Entities/Book.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Task5._2.Entities
+{
+    public class Book
+    {
+        public string Title { get; set; }
+        public DateTime? PublicationDate { get; set; }
```

Comment:
The set of authors of the book (a collection of non-repeating strings, possibly empty).-> to use list is not a good idea

Reply:
Fixed